src/batterystatus.h

```diff
@@ -47,6 +47,8 @@ class SYSTEMSETTINGS_EXPORT BatteryStatus : public QObject
 
     Q_PROPERTY(ChargingMode chargingMode READ chargingMode WRITE setChargingMode
             NOTIFY chargingModeChanged)
+    Q_PROPERTY(bool chargingOverride READ chargingOverride WRITE setChargingOverride
+            NOTIFY chargingOverrideChanged)
```

Comment:
I would prefer e.g. `chargingForced` as the name. More obvious what it means and it would also align better with mce's API.

Reply:
MM, yes, looks like there is some jitter with naming. Mostly coming from: you can't really force charging (say, if charger is not connected,  kernel/whatnot decides that temperature is too high, and so on). But you can override policy that is denying charging when charger is present.

Then again, overall forced-charging / similar derivate probably is more understandable once one forgets about situations where charger is not connected etc as irrelevant.

Reply:
changed: chargingOverride -> chargingForced